### DISCUSSION TITLE
Atrocore installation guide for windows

### DISCUSSION THREAD

**Discussion by nirav182604** [atrocore/atrocore#279] [Q&A] [answered]
Please share a document for step by step installation guide for windows

**Comment by atrolex** [comment, marked as answer]
PHP is much slower on windows than on linux, so we do not recommend to use windows generally and do not intent to make atrocore software work on windows. There are no cron jobs in windows, without them the system will not work. Maybe it can be solved with some workaround, by still, I would recommend to let this idea go :) Use linux, if you wish, as a virtual machine on your windows, it will save you a lot of time :)

**Reply by nirav182604** [reply]
Ok.. let me give a hit to manage Linux in windows once ..  thanks for the reply :)

**Reply by nirav182604** [reply]
Will it work on Kali?

**Reply by atrolex** [reply]
it should work on any linux distro, also on Kali, just install everything what is needed.i
